Dropdown/Dropdown.module.less

```diff
@@ -31,6 +31,10 @@
 				right: 0;
 				margin: 0;
 			}
+
+			.marquee {
+				margin-right: 42px;
```

Comment:
I think we're going to want this, and possibly that `transition-delay: 0.5s;` rule in a variable. It seems unlikely that 42 will work everywhere for all skins.